packages/azure-functions/Shared/build-config-from-message.js

```diff
@@ -0,0 +1,28 @@
+import path from 'path'
+
+const buildConfigFromMessage = message => {
+  const fileLocation = message.location
+  const fileExtension = path.extname(fileLocation)
+  const fileDirectory = path.dirname(fileLocation)
+  const filename = path.basename(fileLocation, fileExtension)
+
+  const config = {
+    fileConfig: {
+      fileLocation,
+      fileExtension,
+      fileDirectory,
+      filename
+    },
+    containerName: message.containerName
+  }
+
+  return Object.freeze({
+    ...config,
+    signalRMessageConfig: {
+      userId: fileDirectory.substring(0, fileDirectory.indexOf('/')),
+      target: `Processed ${filename}${fileExtension}`
+    }
+  })
+}
+
+export default buildConfigFromMessage
```

Comment:
Other than the containerName attribute this file seems almost like a duplicate of https://github.com/DEFRA/biodiversity-net-gain-service/blob/master/packages/azure-functions/Shared/build-upload-config-from-message.js. Could the changes be included in the original file somehow rather than maintaining  two files that are almost identical?